storm-server/src/main/java/org/apache/storm/daemon/nimbus/Nimbus.java

```diff
@@ -4051,7 +4052,9 @@ public TopologyPageInfo getTopologyPageInfo(String topoId, String window, boolea
                                                            nodeToHost,
                                                            workerToResources,
                                                            includeSys,
-                                                           true); //this is the topology page, so we know the user is authorized
+                                                           true, //this is the topology page, so we know the user is authorized
+                                                           null,
+                                                           owner);
```

Comment:
With this change, the other `aggWorkerStats` function is no longer used by anyone and can be removed.

Reply:
removing unused overload